src/index.js

```diff
@@ -99,6 +99,10 @@ class Superlogin extends EventEmitter {
 			return Promise.reject(response);
 		};
 
+		// clear interceptors from a previous configure call
+		this._http.interceptors.request.handlers = [];
```

Comment:
I had the same feeling, but you need to specify an handler id to eject, which you don't get with the example the `forEach` method. So since the `handlers` wasn't prefixed with `_` I found this an acceptable solution. But maybe, like you suggest in https://github.com/micky2be/superlogin-client/issues/17#issuecomment-243663814 we could recreate the whole Axios in the configure handler. So I'll try that.